config/config.go

```diff
@@ -75,7 +75,7 @@ func NewConfigFromText(text string) (ConfigInterface, error) {
 }
 
 // AddConfig adds a new section->key:value to the configuration.
-func (c *Config) AddConfig(section string, option string, value string) bool {
+func (c *Config) AddConfig(section, option, value string) bool {
```

Comment:
I don't think removing duplicated types here would be better. Is this the standard way for Go projects to omit types, or is it used by Google or Golang official team?